packages/utils-debugging-protocol-common/src/debugging-protocol-connector.ts

```diff
@@ -63,20 +65,22 @@ export class Connector implements IConnector {
     private _redirects = new RedirectManager();
     /** A collection of requests with their initial data. */
     private _pendingResponseReceived: Array<Function>;
+    /** Collection of */
+    private _finishedRequests: Map<string, { reject: Function; resolve: Function }>;
     /** List of all the tabs used by the connector. */
     private _tabs = [];
     /** Tells if a favicon of a page has been downloaded from a link tag. */
     private _faviconLoaded: boolean = false;
     /** The amount of time before an event is going to be timedout. */
     private _timeout: number;
     /** Browser PID */
-    private pid: number;
+    private _pid: number;
     private _targetNetworkData: NetworkData;
     private launcher: ILauncher;
```

Comment:
Any reason this isn't `_launcher` since you've added leading underscores to the other private properties?

Reply:
Probably a leftover from when I was cherry-picking. It should have a `_`